SimG4Core/Application/python/NeutronBGforMuons_cff.py

```diff
@@ -5,13 +5,13 @@ def neutronBG(process):
   # common fragment allowing to simulate neutron background in muon system
 
   if hasattr(process,'g4SimHits'):
-  # time window 100 millisecond
+  # time window 10 second
     TimeCut = cms.double(10000000000.0)
     process.common_maximum_time.MaxTrackTime = TimeCut
     process.common_maximum_time.DeadRegions = cms.vstring()
     # Physics List XS
     process.g4SimHits.Physics.type = cms.string('SimG4Core/Physics/FTFP_BERT_XS_EML')
-    process.g4SimHits.Physics.CutsOnProton  = cms.untracked.bool(True)
+    process.g4SimHits.Physics.CutsOnProton  = cms.untracked.bool(False)
```

Comment:
David, cutOnProton is an addition of production threshold for nuclear recoil in elastic scattering processes. It can be set to zero if the flag is False. In that case, all low-energy recoil nuclei will be tracked, that may be not very critical for background neutrons but seems more correct and this is not cost too much.